src/en/proposals/emogarbage-celestial-war-gamemode.md

```diff
@@ -0,0 +1,327 @@
+
+## Gods
+The Gods are invincible observers that serve as the info source of each church. 
+They can not be seen or heard by players, except for other Gods.
+However, in exchange, they have a variety of abilities they can use in order to aid their followers in eventually completing the ascension ritual.
+
+**Divinity** serves as a God's primary resource and is required in order to use their powers.
+Gods themselves cannot create Divinity: it can only come from followers or from things happening on station.
+In this sense, Divinity can come almost passively for some gods, due to how frequently some actions occur.
+
+Divinity can be generated in a few ways:
+- **Preaching:** Followers preaching from a grimoire of their God will create a small amount of divinity for every non-follower in range.
+- **Altar Conversion:** Converting another church's altar will give you a boost in divinity. 
+It will also decrease their amount of divinity. 
+Be sure not to leave altars undefended.
+- **Opposition Follower Sacrifice:** Sacrificing an opposing church's disciple, prophet, celestial artifact, or the station chaplain will give a significant amount of divinity.
+
+On top of this, every god has two additional methods of generating Divinity.
+The first, and easiest, is themed sacrifices. 
+These are accomplished by simply offering specific items to an altar. 
+Which item it is depends on the God.
+The other method is a _divine task_.
+These tasks are entirely nebulous and dependent on a God.
+
+### Abilities
+While the most interesting abilities vary from god to god, all gods share a few common abilities:
+- **Word of God:** (no cost, ~1 minute cooldown) Send a message to your prophet.
+- **Minor Blessing:** (low cost) Give a temporary speed, health, and armor buff to a target follower.
+- **Minor Punishment:** (low cost) Damage and stun a target follower. 
+Used to punish and make examples out of unruly followers.
+- **Celestial Artifact:** (medium cost) When used on a target altar, spawns a random celestial artifact based on the God.
+- **Major Blessing:** (medium cost) Grants the effects of Minor Blessing to all followers on-screen.
+- **Proselytize:** (high cost) When used on a non-follower near an altar, converts them into a disciple.
+
+Additionally, every God gets 3 unique abilities, 2 medium-cost and moderately strong ones and a single high-cost and extremely destructive attack.
+The actual properties of these attacks vary from God to God, but they serve as simple ways to provide additional aid during battles or during the ritual.
+
+## Pantheon
+### Khull of the Sun
+_A ghastly visage of flame. The details are impossible to make out, save for an expression of rage._
+
+Khull is the eldest of the celestial gods and lays domain over fire.
+Their divinity is raised by burning things; setting fire to objects gives minor amounts of divinity and setting fire to mobs gives a higher amount based on the health of the mob.
```

Comment:
Will this just give divinity on setting something on fire, or for as long as it's burning?